CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution
+
+# How to contribute?
+
+## Steps:
+  
+
+  -  Go to [issues](https://github.com/s-ayush2903/Recipebook/issues) & select one upon which you want to work!
+  
+  -  Fork the [repository](https://github.com/s-ayush2903/Recipebook.git)
+  
+  -  Clone the fork [repo](https://github.com/<your_username>/Recipebook)
+     - git clone https://https://github.com/<your_username>/Recipebook
+    
+  -  Create new branch
+     - git checkout -b <Your-Branch-Name>
+  
+  -  Add code/scripts related to your respective issues.
+     - git add <your-contribution>
+ 
+  -  Add a commit message !
+     - git commit -a -m "<Add a message related to changes you made>"
+
+  -  Push changes
+    - git push origin
+
+  
+  -  Push changes
+    - git push -u origin <name_of_your_branch>
```

Comment:
We won't need this, after considering the trailing comment of mine

Reply:
Got it. Will make the changes as required.